test/tests/nuxeo-basics/run.sh

```diff
@@ -9,7 +9,7 @@ NUXEO_TEST_SLEEP=5
 NUXEO_TEST_TRIES=10
 
 cname="nuxeo-container-$RANDOM-$RANDOM"
-cid="$(docker run -d --name "$cname" "$image")"
+cid="$(docker run -d -e LAUNCHER_DEBUG='-Djvmcheck=nofail' --name "$cname" "$image")"
```

Comment:
I was wondering on if I had to fix the image or not as it's a bug in our code. Meaning if you download the regular ZIP, you have that bug with JVM >= 1.8_u100. I wanted to have the same behavior even if it's a bug.
From a user perspective, I agree that it's better to fix it in the image.